R/facet-.r

```diff
@@ -257,7 +257,7 @@ df.grid <- function(a, b) {
     i_a = seq_len(nrow(a)),
     i_b = seq_len(nrow(b))
   )
-  unrowname(cbind(
+  unrowname(vec_cbind(
     a[indexes$i_a, , drop = FALSE],
     b[indexes$i_b, , drop = FALSE]
   ))
```

Comment:
Might be better to unrowname the individual data frames here so `vec_cbind()` doesn't have to do any "work" to combine them since you are just going to remove them right after that